roles/create-devstack-local-conf/tasks/main.yml

```diff
@@ -211,7 +203,8 @@
       EOF
     executable: /bin/bash
   when:
-    - '"octavia" in enable_services'
+    ('"octavia" in enable_services') and
+    (global_env.OS_BRANCH != 'stable/mitaka' or global_env.OS_BRANCH != 'stable/newton' or global_env.OS_BRANCH != 'stable/queens')
```

Comment:
I'm afraid this condition won't filter any branches, if OS_BRANCH is stable/mitaka, then the first will false, but the second will true, and the whole condition will True.

Reply:
I think @bzhaoopenstack is right, could we use `not in`?

```python
global_env.OS_BRANCH not in ['stable/mitaka','stable/newton','stable/queens']
```

Reply:
yes, `not in` statement looks better